### ISSUE TITLE
Milvus hang when call get_entity_by_id

### ISSUE THREAD

**Issue by del-zhenwu** [milvus-io/milvus#5990]
**Please state your issue using the following template and, most importantly, in English.**

**Describe the bug**
```
2021-06-22 10:37:25,375 - milvus_benchmark.runners.base:149 - INFO - {'total_time': 23.65, 'rps': 42282.59, 'ni_time': 1.18}
2021-06-22 10:37:27,986 - milvus_benchmark.client:49 - DEBUG - Milvus flush run in 2.53s
2021-06-22 10:37:27,988 - milvus_benchmark.client:380 - DEBUG - Row count: 1000000 in collection: <sift_1m_128_l2>
2021-06-22 10:37:27,988 - milvus_benchmark.runners.get:116 - DEBUG - {'collection count': 1000000}
2021-06-22 10:37:27,988 - milvus_benchmark.runners.get:117 - DEBUG - {'flush_time': 2.53}
2021-06-22 10:37:33,214 - milvus_benchmark.client:49 - DEBUG - Milvus load_collection run in 5.23s
2021-06-22 10:37:33,215 - milvus_benchmark.main:89 - INFO - Start run case
2021-06-22 10:37:47,567 - milvus_benchmark.client:49 - DEBUG - Milvus get run in 14.35s
2021-06-22 10:37:47,742 - milvus_benchmark.main:100 - INFO - {'get_time': 14.53}
2021-06-22 10:37:47,742 - milvus_benchmark.main:108 - DEBUG - {'type': 'insert_get_performance', 'value': {'get_time': 14.53}}
2021-06-22 10:37:47,742 - milvus_benchmark.metric.api:29 - DEBUG - {'_version': '0.1', '_type': 'metric', 'run_id': 1624358186, 'mod
e': 'local', 'server': <milvus_benchmark.metrics.models.server.Server object at 0x7f96f869a588>, 'hardware': <milvus_benchmark.metri
cs.models.hardware.Hardware object at 0x7f96f869a518>, 'env': <milvus_benchmark.metrics.models.env.Env object at 0x7f96f869a4e0>, 's
tatus': 'RUN_SUCC', 'err_message': '', 'collection': {'dimension': 128, 'metric_type': 'l2', 'dataset_name': 'sift_1m_128_l2', 'coll
ection_size': 1000000, 'other_fields': None, 'ni_per': 50000}, 'index': {'index_type': 'ivf_sq8', 'index_param': {'nlist': 1024}}, '
search': None, 'run_params': {'ids_length': 1}, 'metrics': {'type': 'insert_get_performance', 'value': {'get_time': 14.53}}, 'dateti
me': '2021-06-22 10:36:26.814692'}
```
**Steps/Code to reproduce behavior**
1. create collection
2. 1m entities inserted (sift-1m-128d)
3. flush and load
4. get entity by id, id length: 1
cost 14s
**Expected behavior**

**Method of installation**
- [ ] Docker/cpu
- [ ] Docker/gpu
- [ ] Build from source

**Environment details**
8c16m 

- Milvus version (master or released version)
standalone version

**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

```yaml
paste-file-content-here
```

**Screenshots**

**Additional context**

**Comment by cydrain** [comment]
Milvus will hang in proxy/task.go L2145
```
func (rt *RetrieveTask) PostExecute(ctx context.Context)
```

**Comment by cydrain** [comment]
Repo: git@github.com:zilliztech/milvus_benchmark.git
Branch: argo

Run this command to re-produce
python main.py --local --suite=suites/2_insert_get.yaml

**Comment by cydrain** [comment]
fixed by #6054

**Comment by czs007** [comment]
> fixed by #6054

@del-zhenwu
